Manatee.Trello.IntegrationTests/ClientTestsOfTheNew.cs

```diff
@@ -21,8 +21,9 @@ public async Task Issue205_CopyCardWithAttachment()
 			var list = board.Lists.Last();
 			var cards = list.Cards;
 			var sourceCard = await TestEnvironment.Current.BuildCard();
-			var jpeg = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Files/smallest-jpeg.jpg");
-			await sourceCard.Attachments.Add(File.ReadAllBytes(jpeg), "smallest-jpeg.jpg");
+			var pdf = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Files/smallest-jpeg.jpg");
```

Comment:
I'd prefer not to change the client tests since they're associated with specific issues.  Can you create a new test that uses the new code path, please?

Reply:
Alright